homeassistant/components/toon/const.py

```diff
@@ -338,6 +340,51 @@
         ATTR_ICON: "mdi:solar-power",
         ATTR_DEFAULT_ENABLED: True,
     },
+    "water_average": {
+        ATTR_NAME: "Average Water Usage",
+        ATTR_SECTION: "water_usage",
+        ATTR_MEASUREMENT: "average",
+        ATTR_UNIT_OF_MEASUREMENT: VOLUME_LHOUR,
+        ATTR_DEVICE_CLASS: None,
+        ATTR_ICON: "mdi:water",
+        ATTR_DEFAULT_ENABLED: True,
+    },
+    "water_average_daily": {
+        ATTR_NAME: "Average Daily Water Usage",
+        ATTR_SECTION: "water_usage",
+        ATTR_MEASUREMENT: "day_average",
+        ATTR_UNIT_OF_MEASUREMENT: VOLUME_CM3,
+        ATTR_DEVICE_CLASS: None,
+        ATTR_ICON: "mdi:water",
+        ATTR_DEFAULT_ENABLED: False,
+    },
+    "water_daily_usage": {
+        ATTR_NAME: "Water Usage Today",
+        ATTR_SECTION: "water_usage",
+        ATTR_MEASUREMENT: "day_usage",
+        ATTR_UNIT_OF_MEASUREMENT: VOLUME_CM3,
+        ATTR_DEVICE_CLASS: None,
+        ATTR_ICON: "mdi:water",
+        ATTR_DEFAULT_ENABLED: True,
+    },
+    "water_meter_reading": {
+        ATTR_NAME: "Water Meter",
+        ATTR_SECTION: "water_usage",
+        ATTR_MEASUREMENT: "meter",
+        ATTR_UNIT_OF_MEASUREMENT: VOLUME_CM3,
```

Comment:
Shouldn't this be M3?

Reply:
I guess that would be a better value to have here.
However, that requires an update of the upstream package.

Reply:
I checked and it's definitely cm³, if not I would have used thousands of liters water a day :)

Reply:
True, but the meter displays a float/comma-separated value of M3.
So converting it to M3 in the upstream package makes more sense I guess.

Reply:
You've marked this as resolved, but it isn't.
Changing the unit of measurement isn't magically going to calculate the right values.
The upstream package needs to be adjusted, released and the version in Home Assistant for the toonapi lib bumped after that.

Reply:
Pull request for toonapi lib: [https://github.com/frenck/python-toonapi/pull/1]https://github.com/frenck/python-toonapi/pull/1)